scripts/update-package-sha.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+NEW_CHECKSUM_STATIC=$(shasum -a 256 Carthage/Sentry.xcframework.zip | awk '{print $1}')
+NEW_CHECKSUM_DYNAMIC=$(shasum -a 256 Carthage/Sentry-Dynamic.xcframework.zip | awk '{print $1}')
+
+os=$(uname)
+
+if [ "$os" == "Linux" ]; then
```

Comment:
Craft pre-release command runs on an ubuntu machine,
`sed` works a little bit different there

Reply:
`m`: Please add this essential info as a comment here.